Resources/Prototypes/_NF/Entities/Structures/Doors/Airlocks/external.yml

```diff
@@ -0,0 +1,24 @@
+- type: entity
+  parent: AirlockExternal
```

Comment:
Looking at this, why not parent from AirlockExternalGlass and only overwrite the sprite?  The rest is identical.

Reply:
Think that was just me being consistent with the way upstream yml is.